js/src/components/google-combo-account-card/index.scss

```diff
@@ -0,0 +1,10 @@
+.google-combo-account-card {
+	.gla-account-card__description {
```

Comment:
@ankitrox I think this won't be necessary if we use the `label` prop of `CheckboxControl`

Reply:
We can't use `label` prop for the reason specified above.